apollo-router/src/plugins/cache/entity.rs

```diff
@@ -539,6 +539,7 @@ async fn cache_lookup_root(
 
     let key = extract_cache_key_root(
         &name,
+        None,
```

Comment:
Question for reviewers: Is there a way to figure out the entity type here?

Reply:
* get the schema at plugin creation (there's a different plugin init function signature for that)
* get the API schema which is a `apollo_copiler::Schema`
* `schema.schema_definition.query.as_ref().map(|q| q.name.to_string())`